### PR TITLE
Add stability tests

### PR DESCRIPTION
**Pull request by trungleduc**
## References
Add `Playwright` tests to test the outputs of notebook with bugged widgets. 

## Code changes
- Add a library of bugged widgets to `tests/widget_lib`
- Separate notebook rendering tests and stability tests in two `Playwright` projects.
- Update github action configuration.

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @trungleduc!

Looks like there is a small conflict to fix.

**Comment by trungleduc** [comment]
@jtpio updated

**Comment by jtpio** [comment]
Looks good!

Left one comment above. Also there is one more conflict now that #1022 has been merged.

**Comment by jtpio** [comment]
> Also there is one more conflict now that #1022 has been merged.

Just rebased to fix the conflict.

**Comment by trungleduc** [comment]
Thank @jtpio, I'm waiting for #984 to be merged before re-working on this PR since it will introduce more conflict to #984. But it seems that we changed the calendar for #984 so this can be merged before.

**Comment by jtpio** [comment]
> we changed the calendar for #984

What does that concretely mean? Would you mind (or @davidbrochart) posting a quick update in https://github.com/voila-dashboards/voila/pull/984 or https://github.com/voila-dashboards/voila/issues/977 if that refers to offline discussions? Thanks!

**Comment by davidbrochart** [comment]
I think #984 is not ready to be merged, but #1025 is.

**Comment by trungleduc** [comment]
This PR is ready again.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -54,6 +54,7 @@ jobs:
       shell: bash -l {0}
       run: |
         pip install "traitlets>=4,<5"
+        pip install ipykernel==5.5.5
```
Comment (jtpio) [inline .github/workflows/main.yml:57]: Just to be sure we don't forget later, where is this issue being tracked?

Comment (trungleduc) [reply .github/workflows/main.yml:57]: Sorry, I forgot about it, where should I put the issue?

**Pull request closed without merging**